### DISCUSSION TITLE
Can't connect to remote camera

### DISCUSSION THREAD

**Discussion by lochermichael** [mrlt8/docker-wyze-bridge#1307] [Q&A]
I have four Wyze Cam v3's, and one of them we travel with to use as a baby monitor. This one won't show up in Docker-Wyze-Bridge when it's away from our home network. I've updated the Docker container and entered the API info. The other cameras work fine. I also tried enabling the P2P mode which didn't help. The camera shows a "satellite dish" icon, not sure if that's good or bad. But when I click on it, it fails to load the stream. Any ideas?

**Comment by mrlt8** [comment]
Unfortunately, this is a known issue on the 2024 firmware. Wyze seems to have changed something that is limiting remote connections.

One option is to downgrade the firmware to something from 2023 or use a VPN to put the bridge on the same network as the cameras with [OpenVPN](https://github.com/mrlt8/docker-wyze-bridge/blob/main/docker-compose.ovpn.yml) or [Tailscale](https://github.com/mrlt8/docker-wyze-bridge/blob/main/docker-compose.tailscale.yml).

**Comment by thereal-BLKMGK** [comment]
I resorted to Tailscale with my remote install. Despite manually downgrading ALL of my cameras to a known older firmware that worked for months one day I began finding my cameras had upgraded - without my input. I also found that when I access some cameras I was presented with a modal window telling me to upgrade and the only choice I had was to allow it or force close the application (IOS). So, I bit the bullet and put a Tailscale install on a router in that location and configure it to allow me to access the camera. Since doing that the streams have been ROCK SOLID. For sure this is a good way to go if you can get a client in that local network to tunnel to!
